--- a/pages/system_basics.c
+++ b/pages/system_basics.c
@@ -40,14 +40,14 @@
     while(IGS023_SCANLINE_RAW() < 120)
     {
         //VRAM->bg[x].code = 0xffff;
-        PALRAM->fg[x] = 0xffff;
-        PALRAM->fg[x+1] = 0xffff;
-        PALRAM->fg[x+2] = 0xffff;
-        PALRAM->fg[x+3] = 0xffff;
-        PALRAM->fg[x+4] = 0xffff;
-        PALRAM->fg[x+5] = 0xffff;
-        PALRAM->fg[x+6] = 0xffff;
-        PALRAM->fg[x+7] = 0xffff;
+        PALRAM->fg[0].colors[x] = 0xffff;
+        PALRAM->fg[0].colors[x+1] = 0xffff;
+        PALRAM->fg[0].colors[x+2] = 0xffff;
+        PALRAM->fg[0].colors[x+3] = 0xffff;
+        PALRAM->fg[0].colors[x+4] = 0xffff;
+        PALRAM->fg[0].colors[x+5] = 0xffff;
+        PALRAM->fg[0].colors[x+6] = 0xffff;
+        PALRAM->fg[0].colors[x+7] = 0xffff;
         x = x ? 0 : 1;
     } 
     frame_count++;
